sources/Outputs/src/Job.cpp

```diff
@@ -46,7 +46,7 @@ namespace dfl {
 namespace outputs {
 
 const std::chrono::seconds Job::timeStart_{0};
-const std::chrono::seconds Job::durationSimu_{30};
+const std::chrono::seconds Job::durationSimu_{100};
```

Comment:
Was this discussed with Marco and Quentin?

Reply:
Yes. this modification is due to comparaison between results on IEEE14 python and DFL. 30s was not enough to reach the asymptot